src/app-layout/index.tsx

```diff
@@ -141,12 +147,49 @@ const OldAppLayout = React.forwardRef(
       { componentName: 'AppLayout', controlledProp: 'toolsOpen', changeHandler: 'onToolsChange' }
     );
 
+    const [activeDrawerId, setActiveDrawersId] = useControllable(
+      drawers?.activeDrawerId,
+      drawers?.onChange,
+      isMobile ? false : tools ? defaults.toolsOpen : '',
+      {
+        componentName: 'AppLayout',
+        controlledProp: 'activeDrawerId',
+        changeHandler: 'onChange',
+      }
+    );
+
+    const toolsItem = {
+      id: 'tools',
+      content: tools,
+      ariaLabels: {
+        triggerButton: 'View tools',
+        closeButton: 'Close tools',
+        content: 'Tools',
+      },
```

Comment:
Does this text suppress the value coming from `ariaLabels` prop?

Reply:
Only for the Tools panel, since this won't be something that is user-controlled. When tools and drawers are both used, I create a tools item to append to the front of the drawers array.

Reply:
If we support auto-adoption of the `tools` properties into `drawers` array it should adopt the values from `ariaLabels` too.

Reply:
Note from live conversations: use the existing `ariaLabels` prop to pull these in, instead of hard-coding them.